src/frontend/src/pages/WorkPackageForm/WorkPackageFormView.tsx

```diff
@@ -160,45 +187,25 @@ const WorkPackageFormView: React.FC<WorkPackageFormViewProps> = ({
     >
       <Box mb={-1}>
         <PageBreadcrumbs
-          currentPageTitle={getTitleFromFormType(formType, wbsElement)}
-          previousPages={[
-            formType !== WPFormType.EDIT
-              ? { name: 'Change Requests', route: routes.CHANGE_REQUESTS }
-              : { name: 'Projects', route: routes.PROJECTS },
-            formType !== WPFormType.EDIT && crIdDisplay
-              ? {
-                  name: `Change Request #${crIdDisplay}`,
-                  route: `${routes.CHANGE_REQUESTS}/${crIdDisplay}`
-                }
-              : {
-                  name: `${projectWbsNamePipe(wbsElement)}`,
-                  route: `${routes.PROJECTS}/${projectWbsPipe(wbsElement.wbsNum)}`
-                }
-          ]}
+          currentPageTitle={`${wbsPipe(wbsElement.wbsNum)} - ${wbsElement.name}`}
+          previousPages={breadcrumbs}
         />
       </Box>
       <PageLayout
         stickyHeader
-        title={getTitleFromFormType(formType, wbsElement)}
-        chips={
-          formType === WPFormType.CREATEWITHCR && (
-            <Tooltip
-              title={
-                'This form will create a change request that when accepted will automatically create a new Work Package'
-              }
-              placement="right"
-            >
-              <HelpIcon style={{ fontSize: '1.5em', color: 'lightgray' }} />
-            </Tooltip>
-          )
-        }
+        title={`${wbsPipe(wbsElement.wbsNum)} - ${wbsElement.name}`}
```

Comment:
Not at my laptop so apologies if u added this and I missed it, can u add this info bubble with tooltip somewhere that makes sense, I want ppl to be able to figure out how to use this form with no ambiguity with just the info on the page without having to ask us.

You can also just delete this code u commented out if it's somewhere else